deploy/test/config/openshift/secrets-access-role.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
```

Comment:
Why do we need a duplication of all the `yaml`s?
Is there a difference between them?

Reply:
there are yamls for helm and for init container. The init container used a different method for substitution that I wasn't intuitive `sh.yml` and Helm uses sed so that I can dynamically update true yaml files

Reply:
please, explain what are those yamls for init container and why do you need to change them in this version

Reply:
see above explanation. .sh.yml is less intuitive and changing this would mean refactoring the codebase and a separate PR